### PR TITLE
Trade account provider

### PR DESCRIPTION
**Pull request by nuel77**
- common wallet-like interface for local accounts in the browser, used as trading accounts in polkadex.

### CODE REVIEW

File: packages/trade-wallet/package.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@polkadex/trade-wallet",
+  "main": "./dist/index.js",
+  "module": "./dist/index.mjs",
+  "types": "./dist/index.d.ts",
+  "license": "GPL-3.0-only",
+  "version": "0.0.2",
```
Comment (serhii-temchenko) [inline packages/trade-wallet/package.json:7]: Isn't it supposed to be "0.0.1"?

File: packages/types/package.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@polkadex/types",
+  "main": "./dist/index.js",
+  "module": "./dist/index.mjs",
+  "types": "./dist/index.d.ts",
+  "license": "GPL-3.0-only",
+  "version": "0.0.2",
```
Comment (serhii-temchenko) [inline packages/types/package.json:7]: Isn't it supposed to be "0.0.1"?

**Review by ap211unitech — APPROVED** [review]

File: packages/react-providers/src/UserAccountsProvider/provider.tsx [hunk 3]
```diff
@@ -0,0 +1,42 @@
+import { createContext, useEffect, useState } from "react";
+import { TradeWallet } from "@polkadex/trade-wallet/src";
```
Comment (minikas) [inline packages/react-providers/src/UserAccountsProvider/provider.tsx:2]: I think 'src' is unnecessary here

**Review by minikas — CHANGES_REQUESTED** [review]

**Review by minikas — APPROVED** [review]
LGTM

**Pull request merged**